tachyon/math/polynomials/univariate/radix2_evaluation_domain.h

```diff
@@ -322,28 +334,34 @@ class Radix2EvaluationDomain : public UnivariateEvaluationDomain<F, MaxDegree>,
       }
     }
 
-    roots_vec_[this->log_size_of_group_ - 1] = std::move(largest);
-    inv_roots_vec_[0] = std::move(largest_inv);
+    {
+      TRACE_EVENT("Subtask", "PrepareRootsVec");
 
-    // Prepare space in each vector for the others.
-    size_t size = this->size_ / 2;
-    for (size_t i = 1; i < this->log_size_of_group_; ++i) {
-      size /= 2;
-      roots_vec_[this->log_size_of_group_ - i - 1].resize(size);
-      inv_roots_vec_[i].resize(size);
-    }
+      roots_vec_[this->log_size_of_group_ - 1] = largest;
+      inv_roots_vec_[0] = largest_inv;
```

Comment:
Why are the `std::move`s removed?

Reply:
Bad conflict resolve. will add them back.